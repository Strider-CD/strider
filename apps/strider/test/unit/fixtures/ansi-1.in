  SettingsManager, when initialized
    with a single item
      â—¦ should retrieve: [2K[0G      âœ“ should retrieve 
      â—¦ should set & retrieve: [2K[0G      âœ“ should set & retrieve 
    with a nested item
      â—¦ should be accessible as a.b: [2K[0G      âœ“ should be accessible as a.b 
      â—¦ should inherit type when not specified: [2K[0G      âœ“ should inherit type when not specified 
      â—¦ should not inherit type when specified: [2K[0G      âœ“ should not inherit type when specified 
      â—¦ should only include settings in the json: [2K[0G      âœ“ should only include settings in the json 
    with multiple items
      â—¦ should serialize: [2K[0G      âœ“ should serialize 
      â—¦ should load: [2K[0G      âœ“ should load 
      â—¦ should getList: [2K[0G      âœ“ should getList 
      â—¦ should getHash: [2K[0G      âœ“ should getHash 
      â—¦ should getHaskKeys: [2K[0G      âœ“ should getHaskKeys 
    with a sub
      â—¦ should add things namespaced: [2K[0G      âœ“ should add things namespaced 
      â—¦ should getList namespaced: [2K[0G      âœ“ should getList namespaced 


  13 passing (7 ms)

