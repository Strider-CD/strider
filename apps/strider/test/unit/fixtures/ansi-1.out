  SettingsManager, when initialized
    with a single item
      ✓ should retrieve 
      ✓ should set &amp; retrieve 
    with a nested item
      ✓ should be accessible as a.b 
      ✓ should inherit type when not specified 
      ✓ should not inherit type when specified 
      ✓ should only include settings in the json 
    with multiple items
      ✓ should serialize 
      ✓ should load 
      ✓ should getList 
      ✓ should getHash 
      ✓ should getHaskKeys 
    with a sub
      ✓ should add things namespaced 
      ✓ should getList namespaced 


  13 passing (7 ms)

